lib/commands/exam.js

```diff
@@ -258,9 +294,20 @@ module.exports = TestCommand.extend({
         testemEvents.recordFailedBrowserId(browserId);
       }
 
-      browserExitHandler();
+      browserExitHandler(true);
     };
 
+    this._addCustomBrowserSocketEvents(events, browserExitHandler, browserFailureHandler);
+  },
+
+  /**
+   * Add browser socket events are needed for both with load-balance and without load-balance
+   *
+   * @param {Object} events
+   * @param {Object} browserExitHandler
+   * @param {Object} browserFailureHandler
+   */
+  _addCustomBrowserSocketEvents(events, browserExitHandler, browserFailureHandler) {
```

Comment:
`_addBrowserSocketEvents` and `_addCustomBrowserSocketEvents` sound a bit too similar... it make me think, what is the difference?
could we rename this to `_getModuleMetadataAndBrowserExitSocketEvents` ?